File: Searching Algorithm/Linear search.c
#include<stdio.h>

int main()
{
	int arr[100], i, num, n, c=0, pos;
	printf("Enter the array size : ");
	scanf("%d", &n);
	printf("Enter the Array Elements : ");
	for(i=0; i<n; i++)
	{
		scanf("%d",&arr[i]);
	}
	printf("Enter the number to be searched: ");
	scanf("%d", &num);
	for(i=0; i<n; i++)
	{
		if(arr[i]==num)
		{
			c=1;
			pos=i+1;
			break;
		}
	}
	if(c==0)
	{
		printf("Number cannot be found..!!");
	}
	else
	{
		printf("%d found at the position %d", num, pos);
	}
	
	return 0;
}

File: Searching Algorithm/Linear Search.py
#commit by Suyash
def linearsearch(char_list, findChar):
	for i in range(len(char_list)):
		if char_list[i] == findChar:	#checking if element exixts or not
			return i+1
	return -1
character=input("Enter numbers/characters/words separated by space: ")
character_list=character.split()
print(character_list)
find_char=input("Enter number/character/word to search: ")
flag=linearsearch(character_list,find_char)
if(flag==-1):
	print("Character not present!")
else:
	print("Character found at position "+str(flag))
